scanner/scanner.go

```diff
@@ -358,15 +373,16 @@ func (s *Scanner) Scan(ctx context.Context, foundCert func(*ct.LogEntry), foundP
 	close(jobs)
 	matcherWG.Wait()
 
-	s.Log(fmt.Sprintf("Completed %d certs in %s", atomic.LoadInt64(&s.certsProcessed), humanTime(int(time.Since(startTime).Seconds()))))
+	s.Log(fmt.Sprintf("Completed %d certs in %s", atomic.LoadInt64(&s.certsProcessed), humanTime(time.Since(startTime))))
 	s.Log(fmt.Sprintf("Saw %d precerts", atomic.LoadInt64(&s.precertsSeen)))
 	s.Log(fmt.Sprintf("%d unparsable entries, %d non-fatal errors", atomic.LoadInt64(&s.unparsableEntries), atomic.LoadInt64(&s.entriesWithNonFatalErrors)))
+
 	return nil
 }
 
-// NewScanner creates a new Scanner instance using client to talk to the log,
-// taking configuration options from opts.
-func NewScanner(client *client.LogClient, opts ScannerOptions) *Scanner {
+// New creates a Scanner instance using client to talk to the log, taking
+// configuration options from opts.
+func New(client *client.LogClient, opts ScannerOptions) *Scanner {
```

Comment:
I suspect Travis is about to tell you that there are a couple of other users of this API that will need migrating...